_posts/2022-03-16-cockpit-265.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Cockpit 265
+author: skobyda
+date: '2022-03-16'
+tags: cockpit, machines
+slug: cockpit-265
+category: release
+summary: 'Animated lists, x-forwarder-for support, crypto policies, disk serial number, metrics showing busiest CPU core, manifest overrides location'
```

Comment:
Seems a little longish, but then again we don't actually show these summaries anywhere. But it's X-Forwarded-For, not lower-case and forwarde*r*